### PR TITLE
Add support link

### LINKED ISSUE #1671: Add v1 of "help"
Currently, our only formal channel to provide support to users is exclusively through the support email. In this format, someone has to view each email and manually assess its level of urgency, if it can be taken care of by anyone on the team (versus engineering), etc. This results in a slower-than-desired response time for users and an inefficient workflow for our team. We believe there are better ways for us to promote and provide support to users and better ways for us to elevate self-service options for frequently experienced issues. 

In this first version we want to:

1. Add a way to access help from the homepage and in the app
2. Support three paths for help (elevate a bug, direct to password resources, submit something else)

The way we create this solution should make it easy to identify high priority issues in the support inbox and help users give us enough information to fully understand / see the problem they are facing.

### LINKED ISSUE #1694: Add Support to Application Insights
We are adding an icon for users to access support directly from the app. We want to be able to track this and see how many users are accessing it from there.

### PR DESCRIPTION
**Pull request by maya**
## Related Issue or Background Info

Closes #1671. Closes #1694. 

### TODO

- [x] get test coverage up

## Add'l info
I usually don't use `target="none"` to open in a new window, but assuming we don't want them to lose context so using.

## Screenshots / Demos
Screen Shot 2021-05-25 at 10 46 21 AM

### UI
- [ ] Any changes to the UI/UX are approved by design

### CODE REVIEW

File: frontend/src/app/commonComponents/Header.tsx [hunk 1]
```diff
@@ -16,6 +20,9 @@ import { LinkWithQuery } from "./LinkWithQuery";
 import { TrainingNotification } from "./TrainingNotification";
 
 const Header: React.FC<{}> = () => {
+  const appInsights = useAppInsightsContext();
+  const trackSupport = useTrackEvent(appInsights, "Support", null, false);
```
Comment (timothybest-usds) [inline frontend/src/app/commonComponents/Header.tsx:24]:
```suggestion
  const trackSupport = useTrackEvent(appInsights, "Support", {});
```

File: frontend/src/app/commonComponents/Header.tsx [hunk 2]
```diff
@@ -188,12 +195,20 @@ const Header: React.FC<{}> = () => {
...
-                </li>
               </ul>
             </li>
-
+            <div>
+              <div className="navlink__support">
+                <a
+                  href="https://simplereport.gov/support"
+                  target="none"
+                  onClick={() => trackSupport}
```
Comment (timothybest-usds) [inline frontend/src/app/commonComponents/Header.tsx:205]:
```suggestion
                  onClick={trackSupport}
```

File: frontend/src/app/commonComponents/Header.tsx [hunk 3]
```diff
@@ -308,7 +323,16 @@ const Header: React.FC<{}> = () => {
                     </span>
                   </li>
                   <li className="usa-sidenav__item">{facility.name}</li>
-                  <li className="usa-sidenav__item">
+                  <li className="usa-sidenav__item navlink__support">
+                    <a
+                      href="https://simplereport.gov/support"
+                      target="none"
+                      onClick={() => trackSupport}
```
Comment (timothybest-usds) [inline frontend/src/app/commonComponents/Header.tsx:330]:
```suggestion
                      onClick={trackSupport}
```

**Review by timothybest-usds — COMMENTED** [review]
Added suggestion to fix `ReactEvent` error

**Review by emmastephenson — APPROVED** [review]

**Comment by maya** [comment]
Thanks for the approval, @emmastephenson. Do you know what I need to do to get SonarCloud coverage up so it passes?

**Comment by emmastephenson** [comment]
The test coverage on this one seems a little tricky, because it's asking you to verify that app insights tracks the support click. I can try to add a test for that; I think I've mocked out `app insights` in another test before.

**Pull request merged**